eax/tests/aes128eax.rs

```diff
@@ -4,4 +4,5 @@
 use aes::Aes128;
 use eax::Eax;
 
+#[cfg(feature = "std")]
```

Comment:
I think this can just be...

```suggestion
#[cfg(feature = "alloc")]
```

Reply:
Thanks for the feedback, fixed